mitodl/mitxonline
# data migration for certificate index page

#### Pre-Flight checklist


#### What are the relevant tickets?
Data migration fix for #692 #693 

#### What's this PR do?
Data Migration to create or update slug certificate index page

#### How should this be manually tested?
- Open CMS and check if Certificate Index Page
- Edit this page and go to promote tab
- check that if the slug value is `certificate` So this migration will do nothing for testing you can change the slug value and then run the migration to see the slug value again it should be fixed to `certificate`
- If Certificate Index Page is not in CMS run this migration the page should be created and its slug value should be `certificate`

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)
